### ISSUE TITLE
"Simplified" pathway diagrams for educators.

### ISSUE THREAD

**Issue by DeniseSl22** [wikipathways/wikipathways.org#75]
Have been talking to some local educators here in Maastricht; after I explain WPs, they do like it, but at first it is just to much information to take in on a first glance.

I think we could make some "simplified" versions of key pathways, which we can show to educators (with some graphical visualisations embedded), so they do see the usefulness and the information is not overloading them. These PWs then shouldn't get a tag for data analysis, but a (new to design tag) for educational purposes (so we can also find them easily etc.).

I will work on a example PW first, related to a course I will be tutor for in Nov/Dec this year (and upload it in this issue). Perhaps we can use the project of Jacob again, write up a story with clickable links and show this of. How far is this project in terms of being linkable to WPs website, download of story in powerpoint slides etc? @egonw

**Comment by Chris-Evelo** [comment]
I like the idea of more tags for different types of selections in general. Not just to search but also to create different download sets for different purposes

**Comment by DeniseSl22** [comment]
Yes, will make data analysis also more comprehensible.... if we have a tag to say "drug pathway", "approved for transcriptomics analysis", "disease specific pathway", "approved for metabolomics analysis", "kinetic drug model approved" etc.... Egon and I will write something up for this in the near future .... Don't know if we want to implement some of these already before the WPs summit, or also ask ideas from people there before we add it all. Have also been talking to Kristina on this. We would like some tags to be added only by a select number of people (so probably also the people that are allowed to delete PW). Everyone can still add the tag "approved for data analysis", but in this we we get a higher level of curated pathways as different downloads available for people, showing of the quality of WPs... (and if people just want lots of PWs, they can still select the "approved for data analysis" set of PWs.

**Comment by Chris-Evelo** [comment]
I think it would be nice to have a plan that we can discuss during the summit. This will be quite important for different user communities, and we could ask them for input.
